packages/cli/src/project/handlebars.ts

```diff
@@ -0,0 +1,33 @@
+import * as changeCase from 'change-case';
+import Handlebars from 'handlebars';
+
+import { IProject } from './project';
+
+
+export class HandlebarsTemplate {
+    static _runtimeOptions: Handlebars.RuntimeOptions = {
+        helpers: {
+            capitalize: (text: string) => {
+            if (!text) return '';
+            return changeCase.capitalCase(text);
+            },
+            toPascalCase: (text: string) => {
+            if (!text) return '';
+            return changeCase.pascalCase(text);
+            },
+            toDotCase: (text: string) => {
+            if (!text) return '';
+            return changeCase.dotCase(text);
+            },
+            toKebabCase: (text: string) => {
+            if (!text) return '';
+            return changeCase.kebabCase(text);
+            },
+        },
+    };
+
+    static render(input: any, options?: CompileOptions, project?: IProject) {
+        const template = Handlebars.compile(input, options);
+        return template(project, this._runtimeOptions);
+    }
+}
```

Comment:
does `this` work in static functions?

Reply:
in static functions this refers to the class itself not the instance